### PR TITLE
Add test case for ContextManagerBuilderParameter

### LINKED ISSUE #15214: Add test case for ContextManagerBuilderParameter.isEmpty()

### PR DESCRIPTION
**Pull request by jiangtao69039**
Fixes #15214 .

### CODE REVIEW

File: shardingsphere-mode/shardingsphere-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/ContextManagerBuilderParameterTest.java [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+import java.util.Properties;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.lenient;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
```
Comment (yx9o) [inline shardingsphere-mode/shardingsphere-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/ContextManagerBuilderParameterTest.java:40]: `RunWith` is not required.

File: shardingsphere-mode/shardingsphere-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/ContextManagerBuilderParameterTest.java [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+    @Test
+    public void assertIsEmpty() {
+        Map<String, SchemaConfiguration> mockSchemaConfigs = getMockSchemaConfiguration(true, true);
+        ContextManagerBuilderParameter contextManagerBuilderParameter = ContextManagerBuilderParameter.builder()
+                .globalRuleConfigs(Collections.emptyList())
+                .props(new Properties())
+                .schemaConfigs(mockSchemaConfigs)
+                .build();
+        boolean actualIsEmpty = contextManagerBuilderParameter.isEmpty();
```
Comment (yx9o) [inline shardingsphere-mode/shardingsphere-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/ContextManagerBuilderParameterTest.java:51]: Defining variables is not required, you can directly assert expressions.

**Comment by yx9o** [comment]
@jiangtao69039 Hi, thanks for your contribution. I have some suggestions on code style, and look forward to your correction.

**Comment by jiangtao69039** [comment]
resolved

File: shardingsphere-mode/shardingsphere-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/ContextManagerBuilderParameterTest.java [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+        assertFalse(contextManagerBuilderParameter.isEmpty());
+    }
+
+    private Map<String, SchemaConfiguration> getMockSchemaConfiguration(final boolean dataSourcesIsEmpty, final boolean ruleConfigurationsIsEmpty) {
+        SchemaConfiguration mockSchemaConfiguration = Mockito.mock(SchemaConfiguration.class, RETURNS_DEEP_STUBS);
+        lenient().when(mockSchemaConfiguration.getDataSources().isEmpty()).thenReturn(dataSourcesIsEmpty);
+        lenient().when(mockSchemaConfiguration.getRuleConfigurations().isEmpty()).thenReturn(ruleConfigurationsIsEmpty);
+        Map<String, SchemaConfiguration> schemaConfigs = new HashMap<>();
+        schemaConfigs.put("logic", mockSchemaConfiguration);
+        return schemaConfigs;
```
Comment (menghaoranss) [inline shardingsphere-mode/shardingsphere-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/ContextManagerBuilderParameterTest.java:183]: the return value should be defined as `result`

**Review by menghaoranss — CHANGES_REQUESTED** [review]

**Review by menghaoranss — APPROVED** [review]

**Pull request merged**